.github/workflows/php_tests.yml

```
@@ -62,13 +62,17 @@ jobs:
         uses: shivammathur/setup-php@v2
         with:
           php-version: '8.2'
+      - name: Get WC pinned pnpm version
+        id: get_wc_pnpm_version
+        run: echo "WC_PNPM=$(node -p "require('/tmp/woocommerce/plugins/woocommerce/package.json').engines.pnpm")"  >> $GITHUB_OUTPUT
```

This works but I think the quotes should be escaped. What's happening right now is this (string concatenation):

```
waclaw@Waclaws-MacBook-Pro woocommerce % echo "hey"foo"hey"
heyfoohey
```

How about this instead?

```suggestion
        run: echo "WC_PNPM=$(node -p \"require('/tmp/woocommerce/plugins/woocommerce/package.json').engines.pnpm\")" >> "$GITHUB_OUTPUT"
```